pkg/scheduler/eventhandlers.go

```diff
@@ -203,18 +203,10 @@ func (sched *Scheduler) updatePodInCache(oldObj, newObj interface{}) {
 		klog.ErrorS(nil, "Cannot convert newObj to *v1.Pod", "newObj", newObj)
 		return
 	}
-
-	// A Pod delete event followed by an immediate Pod add event may be merged
-	// into a Pod update event. In this case, we should invalidate the old Pod, and
-	// then add the new Pod.
-	if oldPod.UID != newPod.UID {
-		sched.deletePodFromCache(oldObj)
-		sched.addPodToCache(newObj)
-		return
-	}
+	klog.V(3).InfoS("Update event for scheduled pod", "pod", klog.KObj(oldPod))
```

Comment:
Do we need this?

Reply:
I think so.

Reply:
You are right... every pod healthcheck causes an update, AFAIK. So probably too noisy. Can we make it v(4)?